### DISCUSSION TITLE
Help with a error after trying to change port

### DISCUSSION THREAD

**Discussion by balucanb** [VorlonCD/bi-aidetection#305] [Q&A] [answered]
So I have not had to make any changes in forever because everything has been working fine, Thanks @VorlonCD . I need to change the port AI Tool uses from 81 to 82, really any other port since I need 81 for something else. I thought I could just change it in the application but that does not seem to be working as I am getting errors (sent to Telegram) saying 
ERROR> ...Problem with AI URL: ' Unable to connect to the remote server [WebException] (In AITOOL.GetDetectionsFromAIServer:2102)' - 'http://127.0.0.1:81/v1/vision/detection' (URL ErrCount=1, max allowed of 5) 
and 
ERROR> Unable to connect to the remote server [WebException] (In AITOOL.GetDetectionsFromAIServer:2102)
Obviosly the error is with me- How can I correct and get everything using port 82? Thanks

**Comment by grrrah** [comment, marked as answer]
I had this error recently - not sure same as yours. Two things to check:
1) Settings Tab->Click on Edit button for AI Server URL... make sure the port 81 server instance has been deleted
2) DeepStack tab... make sure the port appears correctly there. Stop and Restart the server

**Reply by balucanb** [reply]
Thanks I am an idiot- the second I hit enter on my post I remembered OH YEAH! settings tab and edit the the stinking entry. :)
